src/index.html

```diff
@@ -13,5 +13,91 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:8080/" method="post">
+
+      <fieldset class="form__box">
```

Comment:
There should be no blank lines between parent and child tags
```suggestion
    <form action="http://localhost:8080/" method="post">
      <fieldset class="form__box">
```